src/Indexer/DocumentIndexer.php

```diff
@@ -262,17 +267,22 @@ private function getIndexer(): Indexer
      * Retrieve the JSON to send to Elasticsearch for search
      *
      * @param null $query
+     * @param int $page
      * @return mixed|string
      * @throws ReadFileException
      */
-    private function getSearch($query = null)
+    private function getSearch($query = null, int $page = 0, $max)
```

Comment:
This is not possible to have the `$max` without default value if the previous argument has one.